docs/index.md

```diff
@@ -161,16 +161,16 @@ See: [github:felixfbecker/php-language-server](https://github.com/felixfbecker/p
 
 #### Intelephense
 
-See [bmewburn/intelephense-server/](https://github.com/bmewburn/intelephense-server/)
+See [bmewburn/intelephense](https://github.com/bmewburn/intelephense)
```

Comment:
```suggestion
See [bmewburn/intelephense-docs](https://github.com/bmewburn/intelephense-docs)
```

Reply:
The author has confirmed it's now closed source.
https://github.com/bmewburn/intelephense/issues/172#issuecomment-465749434

Reply:
Well spotted :) thanks